workspace/lib/lita/handlers/teamcity.rb

```diff
@@ -615,6 +642,58 @@ def format_artifacts_build(build_type, build_id, build_number)
       def is_numeric(number)
         true if Integer(number) rescue false
       end
+
+      def fetch_download_url_by_pr_id(pr_id)
+        site_info = JSON.parse(config.artifactory_info)
+        site_url = site_info['site_url']
+        download_url = site_info['download_url']
+        full_url = "#{site_url}#{download_url}#{pr_id}/"
+        specific_url = "#{pr_id}/"
+        begin
+          data = Nokogiri::XML(open(full_url))
```

Comment:
Can we use  [REST API](https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API) to get download links?